leases/utils.py

```diff
@@ -219,3 +221,60 @@ def terminate_lease(
         )
 
     return lease
+
+
+def exchange_berth_for_lease(
+    old_lease: BerthLease,
+    new_berth: Berth,
+    switch_date: date,
+    old_lease_comment: str,
+    new_lease_comment: str,
+) -> (BerthLease, BerthLease):
+    # Avoid circular imports
+    from leases.models import BerthLease
+
+    end_date = old_lease.end_date
+    contract = None
+    try:
+        contract = old_lease.contract
+    except old_lease.contract.DoesNotExist:
+        pass
+
+    old_lease = terminate_lease(
+        lease=old_lease, end_date=switch_date, profile_token="", send_notice=False,
```

Comment:
You can skip `profile_token` since it's a kwarg :)

```suggestion
        lease=old_lease, end_date=switch_date, send_notice=False,
```